nova/tests/unit/virt/vmwareapi/fake.py

```diff
@@ -997,54 +1019,57 @@ def __init__(self, task_name, state="running", result=None,
 
 def create_host_network_system():
     host_net_system = HostNetworkSystem()
-    _create_object("HostNetworkSystem", host_net_system)
+    _create_object(host_net_system)
 
 
 def create_host_storage_system():
     host_storage_system = HostStorageSystem()
-    _create_object("HostStorageSystem", host_storage_system)
+    _create_object(host_storage_system)
 
 
 def create_host(ds_ref=None):
     host_system = HostSystem(ds_ref=ds_ref)
-    _create_object('HostSystem', host_system)
+    _create_object(host_system)
 
 
 def create_datacenter(name, ds_ref=None):
     data_center = Datacenter(name, ds_ref)
-    _create_object('Datacenter', data_center)
+    _create_object(data_center)
+    return data_center.obj
```

Comment:
Why does this function return something now?

Reply:
I've removed it again.